core/blockchain.go

```diff
@@ -2151,24 +2150,31 @@ func (bc *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 		log.Crit("Failed to delete useless indexes", "err", err)
 	}
 
-	// Collect the logs
+	// Notify about removed logs first.      
+	if len(deletedLogs) > 0 {
```

Comment:
Can we do the same thing to `deletedLogs`?

It's same to `rebirthLogs`, the oldChain is also tracked, we can split the deletedLogs into different batches as well.

Reply:
done